skimage/color/colorconv.py

```diff
@@ -502,14 +502,8 @@ def _convert(matrix, arr):
         The converted array.
     """
     arr = _prepare_colorarray(arr)
-    arr = np.swapaxes(arr, 0, -1)
-    oldshape = arr.shape
-    arr = np.reshape(arr, (3, -1))
-    out = np.dot(matrix, arr)
-    out.shape = oldshape
-    out = np.swapaxes(out, -1, 0)
-
-    return np.ascontiguousarray(out)
+
+    return np.dot(arr, matrix.T.copy())
```

Comment:
@ahojnnes I haven't been able to benchmark it making any difference one way or the other.  `np.dot` is probably a call to some blas function in any decent numpy install, and the blas function would almost certainly read a small matrix just once into registers and then not have to read it from memory (or rearrange it) ever after that... but in case that's not what happens, it would probably be better to have a contiguous array rather than a view.